File: Modificador de 4 lenguaje/js/clasenormales/Computadora.js
class Computadora {
    constructor(marca, ram) {
        this.marca = marca;
        this.ram = ram;
    }

    mostrar() {
        console.log(`Computadora: ${this.marca}, RAM: ${this.ram}GB`);
    }
}

module.exports = Computadora;

File: Modificador de 4 lenguaje/C#/ClaseNormales/ClasesNormales/Computadora.cs
namespace EjemploConstructores.ClasesNormales
{
    class Computadora
    {
        public string Procesador;
        public int RamGb;

        // Constructor sobrecargado (único), ejemplificado con un parámetro int
        // que establece RAM y deja un procesador por defecto.
        public Computadora(int ramGb)
        {
            Procesador = "Desconocido";
            RamGb = ramGb;
        }

        public void Mostrar()
        {
            System.Console.WriteLine($"Computadora: CPU={Procesador}, RAM={RamGb}GB");
        }
    }
}
